### ISSUE TITLE
java.lang.IllegalArgumentException: No enum constant org.bukkit.block.Biome.MUTATED_FOREST

### ISSUE THREAD

**Issue by Scrity1976** [tastybento/askyblock#445]
http://pastebin.com/jH1S45ZM

Spigot: 1.8.9
Config.yml is updated.

**Comment by tastybento** [comment]
You should double check your config.yml. It's in the biomes section:
```
  MUTATED_FOREST:
    icon: RED_ROSE
    friendlyname: "&aFlower Forest"
    description: "Trees and flowers!"
    #permission: any.perm.you.want
```
The old name is FLOWER_FOREST.
Quick fix - update to 1.11.2. ;-)

**Comment by tastybento** [comment]
No response - closing.
